packages/core/src/storage/vectorStore/types.ts

```diff
@@ -49,6 +49,7 @@ export interface VectorStoreQuery {
   queryEmbedding?: number[];
   similarityTopK: number;
   docIds?: string[];
+  nodeIds?: string[];
```

Comment:
Here, docIds and nodeIds are duplicative.

Reply:
If you want, you can switch it globally to nodeIds and remove docIds..

Reply:
@yisding Aren't

https://github.com/run-llama/LlamaIndexTS/blob/main/packages/core/src/Node.ts#L51

The nodeId is a random uuid

The document id is another one, that will be possible to update when document management be added

As well when you break the document in nodes they'll have the same document id but different nodeIds

Reply:
how about moving this out to another PR then?

Reply:
@marcusschiesser looks good, removed it for now